testcrate/tests/addsub.rs

```diff
@@ -44,20 +44,22 @@ mod int_addsub {
                     use compiler_builtins::int::addsub::{$fn_add, $fn_sub};
 
                     fuzz_2(N, |x: $i, y: $i| {
-                        let add0 = x.overflowing_add(y);
-                        let sub0 = x.overflowing_sub(y);
-                        let add1: ($i, bool) = $fn_add(x, y);
-                        let sub1: ($i, bool) = $fn_sub(x, y);
-                        if add0.0 != add1.0 || add0.1 != add1.1 {
+                        let (add0, add_o0)= x.overflowing_add(y);
+                        let (sub0, sub_o0)= x.overflowing_sub(y);
```

Comment:
```suggestion
                        let (add0, add_o0) = x.overflowing_add(y);
                        let (sub0, sub_o0) = x.overflowing_sub(y);
```

Reply:
Thanks. Considering a large portion of this crate's tests lives in macros, I may eventually pull in my rustfmt hack from https://github.com/rust-lang/libc/blob/1f68d578c430a9eaed30ad04e29d7e70ef7ab263/ci/style.sh#L24-L63.